wordle-web/src/components/__tests__/LetterButton.spec.ts

```diff
@@ -8,14 +8,14 @@ describe('LetterButton', () => {
   it('renders properly', () => {
     const letter = new Letter('a')
     const wrapper = mount(LetterButton, { props: { letter } })
-    expect(wrapper.text()).toContain('a')
-    expect(wrapper.attributes('color')).toBe('grey')
+    expect(wrapper.text()).toContain('A')
+    expect(wrapper.attributes('color')).toBe('darkgrey')
```

Comment:
sometimes using strings literals for checking can cause issues. I would recommend using string objects instead

Reply:
yeah, i had it checking letterstatus in my assignment 2 version, but this is their assignment3 stuff. I don't think i should change their stuff? idk, what do you think?

Reply:
kinda up to you. string objects are a better practice imo